src/simpleGit.ts

```diff
@@ -50,9 +50,6 @@ export class SimpleGit extends GitManager {
                 process.env[key] = value;
             }
 
-            const debug = require('debug');
```

Comment:
That's necessary for automated debug logs. https://github.com/steveukx/git-js/blob/main/docs/DEBUG-LOGGING-GUIDE.md

Reply:
@Vinzent03 
OK👍 I added again
3723ca1